### PR TITLE
Fix bugs with the demo + update deps.

### PR DESCRIPTION
**Pull request by darktears**
Fix bugs with the demo not being updated. Simplified the polyfill code
as well.

### CODE REVIEW

**Comment by darktears** [comment]
@kenchris : PTAL

File: src/index.js [hunk 1]
```diff
@@ -100,23 +100,24 @@ export class FoldablesFeature {
...
 
     if (this.spanning === "single-fold-vertical") {
       const width = window.innerWidth / 2 - this.foldSize / 2;
       const height = window.innerHeight;
       return [
-        { top: 0, left: 0, width, height},
-        { top: 0, height, left: width, width: this.foldSize },
-        { top: 0, left: window.innerWidth / 2 + this.foldSize / 2, width, height }
+        { top: 0, left: 0, width: width, height: height},
+        { top: 0, height: height, left: width, width: this.foldSize },
```
Comment (kenchris) [inline src/index.js]: height is the same as height:height - is a JS feature

**Review by kenchris — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (darktears) [reply src/index.js]: didn't work quite well here...

Comment (darktears) [reply src/index.js]: Nevermind, it works.

**Review by kenchris — APPROVED** [review]

**Pull request merged**